force-app/main/default/objects/Template__c/Template__c.object-meta.xml

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
> Tasks can either be entered manually or formed from a Template.

You're missing a relationship between the `Journey` and the `Template` objects, which is going to make the process of forming a Journey from a Template much harder.

Reply:
You were correct to remove that lookup from Template to Journey because it was the wrong type of relationship and in the wrong direction. You needed to replace the relationship, not remove it altogether.